houdini/css_painting_api/example-boxbg.html

```diff
@@ -7,16 +7,16 @@
     <style>
       li {
         background-image: paint(boxbg);
-        --boxColor: hsla(55, 90%, 60%, 1);
+        --boxColor: hsl(55 90% 60% /  1);
```

Comment:
Same here about alpha values of `1`, should we instead use `hsl(55 90% 60%)`?

Reply:
Good catch! I've removed all the opacity `1`  values.